cloudnet-cord/cloudnet-log/src/main/java/eu/cloudnetservice/cloudnet/v2/logging/CloudLogger.java

```
@@ -39,39 +36,68 @@
      * @throws IOException            when creating directories or files in {@code local/}
      *                                was not possible
      */
-    public CloudLogger() throws IOException {
+    public CloudLogger() {
         super("CloudNetServerLogger", null);
 
         if (!Files.exists(Paths.get("local"))) {
-            Files.createDirectory(Paths.get("local"));
+            try {
+                Files.createDirectory(Paths.get("local"));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
         }
         if (!Files.exists(Paths.get("local", "logs"))) {
-            Files.createDirectory(Paths.get("local", "logs"));
+            try {
+                Files.createDirectory(Paths.get("local", "logs"));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
         }
 
         setLevel(Level.ALL);
 
-        this.reader = new ConsoleReader(System.in, System.out);
+        try {
+            this.reader = new ConsoleReader(System.in, System.out);
+        } catch (IOException e) {
+            e.printStackTrace();
```

If the reader cannot get initialized you should throw the exception instead of just catching it (will cause null-problems in the next line)